docs/reference/api-overview.md

```diff
@@ -8,55 +8,62 @@ approvers:
 
 Most operations can be performed through the
-[kubectl](/docs/user-guide/kubectl-overview/) command-line interface or other
-command-line tools, such as [kubeadm](/docs/admin/kubeadm/), which in turn use
-the API. However, the API can also be accessed directly using REST calls.
+[kubectl](/docs/reference/kubectl/overview/) command-line interface or other
+command-line tools, such as [kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/), which in turn use
+the API. However, you can also access the API directly using REST calls.
 
 Consider using one of the [client libraries](/docs/reference/client-libraries/)
 if you are writing an application using the Kubernetes API.
 
 ## API versioning
 
-To make it easier to eliminate fields or restructure resource representations, Kubernetes supports
-multiple API versions, each at a different API path, such as `/api/v1` or
+To eliminate fields or restructure resource representations, Kubernetes supports
+multiple API versions, each at a different API path. For example: `/api/v1` or
 `/apis/extensions/v1beta1`.
 
-The version is set at the API level rather than at the resource or field level to ensure that the API presents a clear, consistent view of system resources and behavior, and to enable controlling access to end-of-life and/or experimental APIs. The JSON and Protobuf serialization schemas follow the same guidelines for schema changes; all descriptions below cover both formats.
+The version is set at the API level rather than at the resource or field level to:
+- Ensure that the API presents a clear and consistent view of system resources and behavior.
+- Enable control access to end-of-life and/or experimental APIs.
 
-Note that API versioning and software versioning are only indirectly related.  The [API and release
-versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md) describes the relationship between API versioning and software versioning.
+The JSON and Protobuf serialization schemas follow the same guidelines for schema changes. The following descriptions cover both formats.
 
-Different API versions imply different levels of stability and support.  The criteria for each level are described
-in more detail in the [API Changes documentation](https://git.k8s.io/community/contributors/devel/api_changes.md#alpha-beta-and-stable-versions).  
+**Note:** the API versioning and software versioning are indirectly related.  The [API and release
+versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md) describes the relationship between API versioning and software versioning.
+{: .note}
 
-The criteria are summarized here:
+Different API versions imply different levels of stability and support. You can find more information about the criteria for each level in the [API Changes documentation](https://git.k8s.io/community/contributors/devel/api_changes.md#alpha-beta-and-stable-versions).  
```

Comment:
s/imply/indicate
(since we're asking you to make other changes anyway. It's also in the original, but not quite accurate there either.)